### PR TITLE
Football + Kabaddi + Bugs 

### LINKED ISSUE #41: Implement Kabaddi Counter.

### LINKED ISSUE #43: Implement Football Counter

### LINKED ISSUE #53: Bug Reported : Canvas:trying to load too large bitmap
Whenever the app was launched there was this bug which caused the app to crash.

### PR DESCRIPTION
**Pull request by Sarthak2601**
Fixes #41  
Fixes #43 
Fixes #53 

**Bug Report -
Bug -- Canvas:trying to load too large bitmap 
Debugged the project and found that one image(Badminton image) in the UI was causing the app to crash again and again. 
Fix -- Changed the badminton image. App working properly now. 

**Football Counter 
New Improved UI 
Red and Yellow card functionality added along with usual reset and finish functions. 

**Kabaddi Counter 
New Improved UI 
New functionalities include - 
1. Bonus Point 
2. All Out 
3. Multi-Point scoring through dialog box. 
Along with usual reset and finish functions. 

**String file updated. 
The Main Activity UI contained hard coded data, so converted that to string resource. 
Kabaddi and Football counters also string resource coded. 

Attachments -- Kabaddi and Football Counters (gifs) 

kab
kab2

### CODE REVIEW

**Comment by Sarthak2601** [comment]
@chanmol1999 Kindly review and please ignore the gif quality. Thanks.

File: Scorecounter/app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -137,15 +141,79 @@
...
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginTop="10dp"
+        android:layout_marginBottom="10dp"
+        android:layout_marginRight="5dp"
+        android:layout_marginLeft="5dp">
+
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="200sp"
```
Comment (chanmol1999) [inline Scorecounter/app/src/main/res/layout/activity_main.xml]: Use resources for these dimensions.

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
Good work till now.
Just add a result summary in Football.
Like how many and which cards to each side etc.

**Comment by Sarthak2601** [comment]
@chanmol1999 
Updated the Alert Dialog box and the resources were used for dimensions. 
Screenshot Attached. 
Screenshot_2019-12-14-13-52-15-898_com example anmol courtcounter

**Review by chanmol1999 — APPROVED** [review]

**Pull request merged**